pro/src/components/pages/Offerers/Offerer/VenueV1/fields/SiretOrCommentFields/__specs__/SiretOrCommentFields.spec.tsx

```diff
@@ -0,0 +1,83 @@
+import '@testing-library/jest-dom'
```

Comment:
tu à trois warning lorsque tu lance ce test: 
```shell
    console.error
      Warning: An update to ForwardRef(Field) inside a test was not wrapped in act(...).
      
      When testing, code that causes React state updates should be wrapped into act(...):
      
      act(() => {
        /* fire events that update state */
      });
      /* assert on the output */
```

note: fixé dans mon commit de proposition `TO SQUASH`